src/test/java/bio/terra/integration/IngestTest.java

```diff
@@ -237,6 +237,22 @@ public void ingestAndUpdateParticipantsWithTransaction() throws Exception {
     assertThat("Committed rows are there", bqQueryResultCommitted.getTotalRows(), equalTo(6L));
   }
 
+  @Test
+  public void ingestJsonData() throws Exception {
+    IngestRequestModel ingestRequest =
+        dataRepoFixtures.buildSimpleIngest(
+            "participant", "ingest-test/ingest-test-participant-with-json-data.json");
+    dataRepoFixtures.ingestJsonData(steward(), datasetId, ingestRequest);
+    List<String> vals =
+        dataRepoFixtures.getColumnValues(steward(), datasetId, "participant", "jsonData", 5);
+    assertThat(
+        "correct jsonData values",
+        vals,
+        containsInAnyOrder(
+            containsString("{\"testField\":\"Bye-bye\"}"),
+            containsString("{\"testField\":\"Hello\"}")));
```

Comment:
I think using `JSONAssert` can simplify this. TDR should already have JSONAssert as a dependency via spring boot.